azure-pipelines.yml

```diff
@@ -58,13 +58,15 @@ jobs:
             /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat)
             /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)
             /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+            /p:BaselineAllAPICompatError=false
```

Comment:
@ericstj the `-ci` switch doesn't specifically set any msbuild properties, so I had to pass this in explicitly

Reply:
Are you sure?  We depend on properties elsewhere, let me dig something up.

Reply:
Arcade should set `ContinuousIntegrationBuild` property, which does appear to be set for standard builds: https://dev.azure.com/dnceng/public/_build/results?buildId=185990&view=logs&jobId=63b49790-6233-5870-8977-1ff42860cf98&taskId=d5ac7b83-d192-58c9-3d0d-9ab0cb98f90f&lineStart=31&lineEnd=31&colStart=240&colEnd=274

Reply:
Ah, looks like you're right. I was only looking at what we pass in `Build.ps1`, but I missed https://github.com/dotnet/arcade/blob/f15f32d01fcf5d7f60f2246faf6db5b337e72f45/eng/common/tools.ps1#L510. Will update the PR.